deploy/config.yaml

```diff
@@ -0,0 +1,26 @@
+---
+ignoredNamespaces: 
```

Comment:
Is this suppose to be how we assume defaults, per your doc update?  If so, I'm not sure that is the best way to do it.

Or is this just a ease of use thing for our development flow via `make deploy` that bypasses the operator?

If so, we should have settings for the other 2 fields, for clarity as much as anything, even if defaults are gleamed in the code itself.

Reply:
Yes, the `config.yaml` is in fact the configuration file embedded in the `ConfigMap`, therefore that's the place to record the default values for OpenShift related things. Other default attributes the operator needs to assume in order to function correctly [is here](https://github.com/otaviof/csi-driver-shared-resource/blob/04372abf4f3b05cf1e01395d64cf87207f8adf61/pkg/config/config.go#L33-L40).

Reply:
https://github.com/otaviof/csi-driver-shared-resource/blob/04372abf4f3b05cf1e01395d64cf87207f8adf61/pkg/config/config.go#L33-L40 was the bit of code I was missing .... thanks @otaviof